package.json

```diff
@@ -26,11 +26,13 @@
   },
   "dependencies": {
     "@cypress/browserify-preprocessor": "^3.0.1",
+    "boolean-parser": "0.0.2",
     "through": "^2.3.8"
   },
   "devDependencies": {
     "@types/browserify": "^12.0.36",
     "@types/chai": "^4.2.22",
+    "@types/cypress": "^1.1.3",
```

Comment:
Cypress ships with types included so this is not required.

https://docs.cypress.io/guides/tooling/typescript-support#Install-TypeScript